src/brogue/RogueMain.c

```diff
@@ -444,7 +444,7 @@ void initializeRogue(uint64_t seed) {
         theItem = addItemToPack(theItem);
 
         theItem = generateItem(STAFF, STAFF_OBSTRUCTION);
-        theItem->enchant1 = 2;
+        theItem->enchant1 = 10;
```

Comment:
There's a way to fold this commit into the previous one (canceling the two inverting diffs) that may be handy here.  (Stop me if you've heard this one before.)

The command is to run `git rebase -i master` while you have your rebalance branch checked out, which will:
* Open your favorite terminal editor (export `EDITOR` first if desired) and show you a list of commits on your branch, one per line
* The left column of each line will be an action to perform on that commit.  The default is `pick`, meaning "keep this commit, unaltered".  There's a legend a the bottom I believe.  Find the line for this "Revert tweak" commit, and change the action to `fixup` (or `f` for short.)
* Save and quit.  git will replay your commits back onto the master branch and perform the requested actions.  Fixing up means merging the commit with its immediate predecessor (and discarding the merged commit's message.  See `squash` to instead merge the two commit messages.)
* If all goes well, `git status` will show that you've diverged from your origin copy of the branch by a few commits.  You will need the force flag to push (`git push -f`), and consumers (like me) would need to "hard reset" to pick up the new version (`git fetch pender && git reset --hard Rebalance-OPstruction pender/Rebalance-OPstruction`)

Rebasing is "dangerous" in the sense that merge conflicts are possible while attempting to apply the patches to a moved master (or while re-ordering your commits, which is another rebase super power.)  However I don't expect it to be a risk in this case.  If it ever does come up, `git rebase --abort` will back out of the whole process and leave you where you started.

Reply:
Thanks, good to know -- although presumably I'll have to learn how to use command line git as a first step for any of this. Currently doing everything via the Mac OS X desktop app and web interface.

Reply:
Oh fair enough, I bet those have a UI to accomplish the same somewhere in them.